File: 100-Days-Series/Day 63/HourglassPattern.js
// Arrow Function to print Hourglass Pattern
const getPrintHourglass=(n)=>{
    // Upper part
    
    for (let i = n; i >= 1; i--) {
        line="";
        for (let space = 1; space <= n - i; space++) {
            line+=" ";
        }
        for (let j = 1; j <= (2 * i - 1); j++) {
            line+="*";
        }
        console.log(line);
    }
    
    // Lower part
    for (let i = 2; i <= n; i++) {
        line="";
        for (let space = 1; space <= n - i; space++) {
            line+=" ";
        }
        for (let j = 1; j <= (2 * i - 1); j++) {
            line+="*";
        }
        console.log(line);
    }
    
}
// Let's Start Here
let rows=6;
getPrintHourglass(rows);

File: 100-Days-Series/Day 63/HourglassPattern.cpp
#include<iostream>
using namespace std;
// Function to print Hourglass Pattern
void getPrintHourglass(int n){
    // Upper part
    for (int i = n; i >= 1; i--) {
        for (int space = 1; space <= n - i; space++) {
            cout<<" ";
        }
        for (int j = 1; j <= (2 * i - 1); j++) {
            cout<<"*";
        }
        cout<<endl;
    }
    // Lower part
    for (int i = 2; i <= n; i++) {
        for (int space = 1; space <= n - i; space++) {
            cout<<" ";
        }
        for (int j = 1; j <= (2 * i - 1); j++) {
            cout<<"*";
        }
        cout<<endl;
    }
}
// Let's Start Here
int main(){
    int rows;
    cout<<"\nEnter the number of rows : ";
    cin>>rows;
    getPrintHourglass(rows);
}